Simple Calculator/Calculator.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Windows.Forms;
+
+namespace Simple_Calculator
+{
+    public partial class Calculator : Form
+    {
+        //variables which are being used
+        string valOne = "";
+        string valTwo = "";
+        string result = "";
+        string ArithOperator;
+        
+        public Calculator()
+        {
+            InitializeComponent();
+        }
+
+        private void button_Click(object sender, EventArgs e)
+        {
+            Button button = (Button)sender;
+            txtResult.Text = txtResult.Text + button.Text;
+        }
+
+        private void operators_Click(object sender, EventArgs e)
+        {
+            valOne = txtResult.Text;
+            Button button = (Button)sender;
+            ArithOperator = button.Text;
+            txtResult.Clear();
```

Comment:
Why do you clear the result field when an operator is clicked?

Reply:
because when i press 22+2,  answer is 224